sencha-workspace/SlateTasksTeacher/app/store/StudentTasks.js

```diff
@@ -0,0 +1,29 @@
+Ext.define('SlateTasksTeacher.store.StudentTasks', {
+    extend: 'Ext.data.Store',
+    requires: [
+        'Slate.cbl.model.StudentTask'
+    ],
+
+    model: 'Slate.cbl.model.StudentTask',
+    remoteFilter: true,
+    remoteSort: true,
+    pageSize: 0,
+
+    proxy: {
+        type: 'slate-records',
```

Comment:
@nbey this class needs to require the proxy class too if it's going to require this `type`. The model could get reconfigured one day to use a different proxy